client/app/models/project.js

```diff
@@ -7,11 +7,12 @@ export default class ProjectModel extends Model {
 
   @attr dcpApplicantCustomerValue;
 
-  @attr({ defaultValue: () => [] }) dcpDcpProjectDcpPackageProject;
+  @hasMany('package', { async: false })
+  packages;
```

Comment:
I think we also need to change this on the backend, in the serializer in `project.controller.ts`.

Reply:
Addressed this comment in the main PR thread, see response there :)